src/scripts/OSFramework/OSUI/Pattern/Progress/AbstractProgressConfiguration.ts

```diff
@@ -9,10 +9,5 @@ namespace OSFramework.OSUI.Patterns.Progress {
 		public Shape: string;
 		public Thickness: number;
 		public TrailColor: string;
-
-		// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
-		constructor(config: any) {
```

Comment:
@rugoncalves just for lack of context why was this removed?

Reply:
Since the constructor is not doing anything, it's the same as not having it (the parent constructor will be invoked).